mlflow/server/js/src/common/utils/Utils.js

```diff
@@ -156,6 +156,18 @@ class Utils {
     return path.replace(/(.*[^/])\.[^/.]+$/, '$1');
   }
 
+  /**
+   * Normalizes a URI, removing redundant slashes and trailing slashes
+   * For example, normalize("foo://bar///baz/") === "foo://bar/baz"
+   */
+  static normalize = (uri) => {
```

Comment:
nit: this doesn't seem to need to be bound to this particular class. can you just do

```js
static normalize(url) {
   ...
}```